spec/acceptance/user_submits_reading_spec.rb

```diff
@@ -17,7 +17,7 @@
     click_button('Submit')
     expect(page).to have_content('123')
     expect(page).to have_content('cpm')
-    expect(user.measurements.size).to eq(1)
```

Comment:
Did you change to `#count` because `#size` did not return `1`? I am curious.

Reply:
Just a habit of optimizing the queries. I reverted the change

Reply:
`#count` always issue query. `#size` issues query if relation is not loaded, and uses `#length` if loaded. If `acting` means `loaded`, you are right. `#length` loads relation if not loaded, and use `#length` to count. 

At this line, I don't think `measurements` relation has not been loaded. That is why I was curious to this change. So, does this example fails unless you change this line?

Reply:
`measurements` loaded and size will return `size` while after `acting` it was updated in db. and yes, spec fails with `size`.